packages/admin-ui-docs/docs/experimental/multi-select-filter.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: MultiSelectFilter
```

Comment:
Honestly, I would call this component: `FilterMultiple`. I did the same on Combobox. It looks smaller. Let me know what you think @Lenderson , @lucasaarcoverde , @anitavincent